### PR TITLE
[IMP] hr_employee_lastnames: rename hooks file

### PR DESCRIPTION
**Pull request by andreagidaltig**
Rename file of hooks to follow guidelines.

### CODE REVIEW

**Comment by andreagidaltig** [comment]
Hi @luisg123v could you review this please?

**Comment by luisg123v** [comment]
@andreagidaltig could you add myself as maintainer, please?

**Comment by andreagidaltig** [comment]
> @andreagidaltig could you add myself as maintainer, please?

I just updated this. Could you review it again, please?

File: hr_employee_lastnames/__manifest__.py [hunk 1]
```diff
@@ -3,7 +3,7 @@
     "name": "HR Employee First Name and Two Last Names",
     "version": "16.0.1.0.1",
     "author": "Vauxoo, Odoo Community Association (OCA)",
-    "maintainer": "Vauxoo",
+    "maintainers": ["Vauxoo", "luisg123v"],
```
Comment (luisg123v) [inline hr_employee_lastnames/__manifest__.py]: `vauxoo` is not actually an expected GitHub username, please remove it.

Comment (andreagidaltig) [reply hr_employee_lastnames/__manifest__.py]: Done.

**Review by luisg123v — APPROVED** [review]
LGTM :+1: 

@moylop260 could you review/merge, please?

**Pull request merged**